pallets/phala/Cargo.toml

```diff
@@ -22,6 +22,7 @@ sp-io = { default-features = false, path = "../../substrate/primitives/io" }
 sp-core = { default-features = false, path = "../../substrate/primitives/core"}
 sp-application-crypto = { default-features = false, path = "../../substrate/primitives/application-crypto" }
 frame-benchmarking = { default-features = false, path = "../../substrate/frame/benchmarking", optional = true }
+node-primitives = { version = "2.0.0", default-features = false, path = "../../substrate/bin/node/primitives" }
```

Comment:
I don't it's used in pallets/phala

Reply:
3d3f2d98aa545d8cfd692a6d19e24c2af7818e01